vNext/extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -393,14 +396,17 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
 
     private createFetchRecord(input?: Request | string, init?: RequestInit): ajaxRecord {
         let id: string;
+        let spanId: string;
         if (this._context && this._context.telemetryTrace && this._context.telemetryTrace.traceID) {
 
             // this format corresponds with activity logic on server-side and is required for the correct correlation
-            id = "|" + this._context.telemetryTrace.traceID + "." + Util.newId();
+            spanId = Util.generateW3CId().substr(0, 16);
+            id = "|" + this._context.telemetryTrace.traceID + "." + spanId;
         } else {
-            id = Util.newId();
+            spanId = Util.generateW3CId();
+            id = spanId;
         }
-        let ajaxData: ajaxRecord = new ajaxRecord(id, this._core.logger);
+        let ajaxData: ajaxRecord = new ajaxRecord(id, this._core.logger, spanId);
```

Comment:
Since `spanId` is useless when trace context is not enabled, this should be made less confusing either moving it somewhere else, etc